site/data/qasida/draft-ya-alim-as-sir-minna

```diff
@@ -0,0 +1,412 @@
+            ["are feared; Your remembrance and Your gratitude"],
+            ["are incumbent, as is Your praise and magnification."],
+
+            ["My Lord, You are my helper."],
+            ["Incite me all the good in me"],
+            ["and make Your gardens my final abode"],
+            ["and seal with faith the lifespans."],
+
+            ["Send blessings in every state"],
+            ["upon he who effaced misguidance,"],
+            ["he to whom the gazelle spoke,"],
+            ["Muhammad, the guide, the leader"],
+
+            ["And all praise is due to Allah along with gratitude"],
+            ["for all His blessings that never cease."],
+            ["We praise Him privately and publicly"],
+            ["by morning and night."],
+
+
+        ]
+    },
+    "order": {
+        "chorus": [
+            [
+                {"repeat": 1, "index": 0 },
+                {"repeat": 1, "index": 1 },
+                {"repeat": 1, "index": 2 },
+                {"repeat": 1, "index": 3 }
+            ]
+        ],
+        "verses": [
+            [
+                {"repeat": 1, "index": 4 },
+                {"repeat": 1, "index": 5 },
+                {"repeat": 1, "index": 6 },
+                {"repeat": 1, "index": 7 }
+            ],
+            [
+                {"repeat": 1, "index": 8 },
+                {"repeat": 1, "index": 9 },
+                {"repeat": 1, "index": 10 },
+                {"repeat": 1, "index": 11 }
+            ],
+            [
+                {"repeat": 1, "index": 12 },
+                {"repeat": 1, "index": 13 },
+                {"repeat": 1, "index": 14 },
+                {"repeat": 1, "index": 15 }
+            ],
+            [
+                {"repeat": 1, "index": 16 },
+                {"repeat": 1, "index": 17 },
+                {"repeat": 1, "index": 18 },
+                {"repeat": 1, "index": 19 }
+            ],
+            [
+                {"repeat": 1, "index": 20 },
+                {"repeat": 1, "index": 21 },
+                {"repeat": 1, "index": 22 },
+                {"repeat": 1, "index": 23 }
+            ],
+            [
+                {"repeat": 1, "index": 24 },
+                {"repeat": 1, "index": 25 },
+                {"repeat": 1, "index": 26 },
+                {"repeat": 1, "index": 27 }
+            ],
+            [
+                {"repeat": 1, "index": 28 },
+                {"repeat": 1, "index": 29 },
+                {"repeat": 1, "index": 30 },
+                {"repeat": 1, "index": 31 }
+            ],
+            [
+                {"repeat": 1, "index": 32 },
+                {"repeat": 1, "index": 33 },
+                {"repeat": 1, "index": 34 },
+                {"repeat": 1, "index": 35 }
+            ],
+            [
+                {"repeat": 1, "index": 36 },
+                {"repeat": 1, "index": 37 },
+                {"repeat": 1, "index": 38 },
+                {"repeat": 1, "index": 39 }
+            ]
```

Comment:
This is perfectly fine don't worry, I should probably add some instructions, but basically each line in the lists above gets a number from 0
![image](https://github.com/adehad/dhikr-hub/assets/26027314/5c3dcd9f-a917-4421-ac2f-24e44332fb30)

So the only change we would need to add here is to add extra entries for the remaining line, and they are grouped by verse sung
```suggestion
            ],
            [
                {"repeat": 1, "index": 40 },
                {"repeat": 1, "index": 41 },
                {"repeat": 1, "index": 42 },
                {"repeat": 1, "index": 43 }
            ],
            [
                {"repeat": 1, "index": 44 },
                {"repeat": 1, "index": 45 },
                {"repeat": 1, "index": 46 },
                {"repeat": 1, "index": 47 }
            ],
            [
                {"repeat": 1, "index": 48 },
                {"repeat": 1, "index": 49 },
                {"repeat": 1, "index": 50 },
                {"repeat": 1, "index": 51 }
            ],
            [
                {"repeat": 1, "index": 52 },
                {"repeat": 1, "index": 53 },
                {"repeat": 1, "index": 54 },
                {"repeat": 1, "index": 55 }
            ],
            [
                {"repeat": 1, "index": 56 },
                {"repeat": 1, "index": 57 },
                {"repeat": 1, "index": 58 },
                {"repeat": 1, "index": 59 }
            ],
            [
                {"repeat": 1, "index": 60 },
                {"repeat": 1, "index": 61 },
                {"repeat": 1, "index": 62 },
                {"repeat": 1, "index": 63 }
            ],
            [
                {"repeat": 1, "index": 64 },
                {"repeat": 1, "index": 65 },
                {"repeat": 1, "index": 66 },
                {"repeat": 1, "index": 67 }
            ],
            [
                {"repeat": 1, "index": 68 },
                {"repeat": 1, "index": 69 },
                {"repeat": 1, "index": 70 },
                {"repeat": 1, "index": 71 }
            ],
            [
                {"repeat": 1, "index": 72 },
                {"repeat": 1, "index": 73 },
                {"repeat": 1, "index": 74 },
                {"repeat": 1, "index": 75 }
            ],
            [
                {"repeat": 1, "index": 76 },
                {"repeat": 1, "index": 77 },
                {"repeat": 1, "index": 78 },
                {"repeat": 1, "index": 79 }
            ],
            [
                {"repeat": 1, "index": 80 },
                {"repeat": 1, "index": 81 },
                {"repeat": 1, "index": 82 },
                {"repeat": 1, "index": 83 }
            ],
            [
                {"repeat": 1, "index": 84 },
                {"repeat": 1, "index": 85 },
                {"repeat": 1, "index": 86 },
                {"repeat": 1, "index": 87 }
            ]
```